docs/resources/job.md

```diff
@@ -343,7 +343,7 @@ This block describes the queue settings of the job:
 
 * `pause_status` - (Optional) Indicate whether this trigger is paused or not. Either `PAUSED` or `UNPAUSED`. When the `pause_status` field is omitted in the block, the server will default to using `UNPAUSED` as a value for `pause_status`.
 * `file_arrival` - (Optional) configuration block to define a trigger for [File Arrival events](https://learn.microsoft.com/en-us/azure/databricks/workflows/jobs/file-arrival-triggers) consisting of following attributes:
-  * `url` - (Required) string with URL under the Unity Catalog external location that will be monitored for new files. Please note that have a trailing slash character (`/`).
+  * `url` - (Required) string with URL under the Unity Catalog external location that will be monitored for new files. Please note that the URL must have the trailing slash character (`/`).
```

Comment:
The [REST API docs](https://docs.databricks.com/api/workspace/jobs/create#trigger-file_arrival) seem to be more clear about this field:

```suggestion
  * `url` - (Required) URL to be monitored for file arrivals. The path must point to the root or a subpath of the external location. Please note that the URL must have a trailing slash character (`/`).
```

The REST API docs don't mention the trailing slash character, is it still required?

Reply:
@gaborratky-db if it's not specified, job creation/update will fail with the error:

```
│ Error: cannot update job: Invalid file arrival trigger configuration URL: must end with '/'
```

So we need to fix REST API spec as well